packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusable.steps.ts

```diff
@@ -1,7 +1,5 @@
-import { cardClassName } from '@fluentui/react-northstar';
-
 const config: ScreenerTestsConfig = {
-  steps: [builder => builder.focus(`.${cardClassName}`).snapshot('Focus on a card')],
+  steps: [(builder, keys) => builder.keys('body', keys.tab).snapshot('Focus on a card')],
```

Comment:
Is this craft? Was this intended?

Reply:
this was unstable and when focus comes by TAB key then it is stable 
Agreed with @pompomon  on it :)